### ISSUE TITLE
Error when trying to open public folder in Finder on MacOS

### ISSUE THREAD

**Issue by matekdk** [keybase/client#12286]
<img width="548" alt="keybase_bug" src="https://user-images.githubusercontent.com/1641909/41195661-42f4d06a-6c32-11e8-8914-0f061fe66fed.png">

my log id: f064258319bcd6214a84571c

**Comment by strib** [comment]
Unfortunately this is either a macOS or an osxfuse bug, we're not sure:

* osxfuse/osxfuse#478
* osxfuse/osxfuse#421

It seems to be pretty rare and goes away after a restart I think.

**Comment by matekdk** [comment]
Well, unfortunately it does not go away after restart on my machine. 

Log id from just after restart: 8824a2001cd4ac0f1de1211c

**Comment by strib** [comment]
Hmm ok, thanks.  Unfortunately Keybase logs don't help much with this, since the error is coming from another layer before the request even gets to us.  Are you able to navigate directly to the folder in Finder, and/or using `cd` on the terminal?

**Comment by matekdk** [comment]
Yes, I can - everything is fully functional from commandline - navigating to the folder, copying files, etc. So this bug is not critical.
